tools/container/containerd/containerd.go

```diff
@@ -187,8 +188,14 @@ func main() {
 // Setup updates a containerd configuration to include the nvidia-containerd-runtime and reloads it
 func Setup(c *cli.Context, o *options) error {
 	log.Infof("Starting 'setup' for %v", c.App.Name)
+	var cliArgs []string
+	if o.HostRootMount != "" {
+		cliArgs = append(cliArgs, "chroot", o.HostRootMount)
+	}
+	cliArgs = append(cliArgs, "containerd", "config", "dump")
 
 	cfg, err := containerd.New(
+		containerd.WithReference(config.FromCLI(cliArgs)),
```

Comment:
Do we allow users to opt-out of this behaviour?

Reply:
I've made the toml config source configurable, so that would be a way for users to opt-out of the default behaviour